### PR TITLE
what-is-near-protocol_2

### PR DESCRIPTION
**Pull request by marco-sundsk**
waiting for review.

### CODE REVIEW

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+要查看日益增长的NEAR上的应用列表，[点击这里](https://awesomenear.com/categories/dapps/)。
+
+轻松开启NFT之旅 - 购买/发布/交易 艺术品 就上 [https://paras.id/](https://paras.id/)
+
+加入BerryClub参与艺术品的创作：[https://berryclub.io/](https://berryclub.io/)
+
+# 使用NEAR的费用如何（Gas）？
+当访问NEAR网络更新和改变数据时，运行网络基础架构的人就产生了一些开销。某些地方的某些计算机运行你的请求，而运营这些计算机的验证人需要支付显著的费用以确保这些计算机的运行。
+
+与其他可编程网络类似，NEAR通过交易费，也叫Gas费补偿这些人。
```
Comment (MartinYeung5) [inline cn/resources/guides/what-is-near-protocol_2.md:60]:
```suggestion
与其他可编程的区块链类似，NEAR通过交易费，也叫Gas费补来偿验证者。
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+如果你熟悉web2上的云服务提供商（亚马逊云，谷歌云等），区块链与其的最大不同是，用户在访问应用的时候直接支付费用，而不是开发者预付所有的基础设施费用。这带来了新的可能性，例如，应用本身规避了长期的入不敷出的资金风险。但这也带来了一些用户可用性上的问题。为了解决这个问题，NEAR也提供了让开发者支付用户使用开销的能力，让使用体验更靠近web2。
```
Comment (MartinYeung5) [inline cn/resources/guides/what-is-near-protocol_2.md:62]:
```suggestion
如果你熟悉web2上的云服务提供商（亚马逊云，谷歌云等），区块链与其的最大不同是，用户在访问应用的时候直接支付费用，而不是开发者预付所有的基础设施费用。这带来了新的可能性，例如，应用本身规避了长期的入不敷出的资金风险。但这也带来了一些用户可用性上的阻碍。为了解决这个问题，NEAR也提供了让开发者支付用户使用开销的能力，让使用体验更靠近web2。
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+**考虑gas时要牢记的两个概念:**
+
+1. Gas单位: 在内部，交易费不是直接用NEAR通证计算的，而是通过一个居中的gas单位。gas单位的好处是，他们是确定的，相同的交易总是消耗相同数量的gas单位。
+2. Gas价格: gas单位乘以gas价格得到具体的向用户收取的费用。这个价格会在每个块时根据网络需求自动重新计算（如果前面块的gas数量超过半满，则价格上升，否则价格下降，每个块的价格变动不会超过1%）。最低的价格由网络配置决定，当前是1亿个yoctoNEAR。
```
Comment (MartinYeung5) [inline cn/resources/guides/what-is-near-protocol_2.md:67]:
```suggestion
2. Gas价格: gas单位乘以gas价格得到具体的向用户收取的费用。这个价格会在每个区块时根据网络需求自动重新计算（如果前面块的gas数量超过半满，则价格上升，否则价格下降，每个区块的价格变动不会超过1%）。最低的价格由网络配置决定，当前是1亿个yoctoNEAR。
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+NEAR的出块时间在一秒左右，这归功于对每个区块gas数量的限制。Gas的单位经过仔细合理的计算，形成了一些易于理解的数字：
+
+10¹² 个gas, 叫 1 TGas (TeraGas)…
+
+≈ 1 毫秒的“计算”时间
+
+…这样一来，在最小1亿个yoctoNEAR每gas的价格下，等值于0.1个milliNEAR。
+
+这里的1ms是一个粗略但有益的近似，也是目前NEAR中单位gas的设计目标。Gas单位不仅封装了计算/CPU时间，也包含带宽/网络时间和存储/IO时间。通过治理机制，将来系统参数是可以调整的，可以改变TGas与1毫秒之间的对应关系。即便如此，上面的内容也是一个不错的开始，让你了解单位gas的意义以及它是怎么得来的。
```
Comment (MartinYeung5) [inline cn/resources/guides/what-is-near-protocol_2.md:81]:
```suggestion
这里的1ms是一个粗略但有用的近似值，也是目前NEAR中对gas单位设定的目标。Gas单位不仅封装了计算/CPU时间，也包含带宽/网络时间和存储/IO时间。通过治理机制，将来系统参数是可以调整的，可以改变TGas与1毫秒之间的对应关系。即便如此，上面的内容也是一个不错的开始，让你了解单位gas的意义以及它是怎么得来的。 
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+|--|--|--|--|
+|创建账户|0.42|0.042|4.2⨉10⁻⁵|
+|转账|0.45|0.045|4.5⨉10⁻⁵|
+|抵押|0.50|0.050|5.0⨉10⁻⁵|
+|添加全权key|0.42|0.042|4.2⨉10⁻⁵|
+|删除Key|0.41|0.041|4.1⨉10⁻⁵|
+
+# 如何获得NEAR?
+## 抵押你的NEAR
+POS模型的关键点在于，社区是如何通过抵押而给予验证人支持的。验证人运营节点以支撑起整个网络，因此获得每年5%固定通胀率的奖励，具体是每个周期（约12个小时）创建出一批新的通证。
```
Comment (MartinYeung5) [inline cn/resources/guides/what-is-near-protocol_2.md:104]:
```suggestion
POS模型的关键点在于，社区是如何通过抵押而给予验证人支持的。验证人运营节点以支撑起整个网络，因此获得每年5%固定通胀率的NEAR通证奖励，具体是每个周期（约12个小时）创建出一批新的通证。
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+抵押:
+
+1. 在导航条上选择 “Staking” (移动端是下拉框)
+2. 点击“Select Validator”按钮
+3. 选择一个人验证人
+4. 确认你的选择并点击“Stake with Validator”
+5. 输入你想要抵押的NEAR数量并点击“Submit Stake”
+
+You will need to confirm two transactions, one to select the validator, and another to deposit and stake with the validator.
```
Comment (MartinYeung5) [inline cn/resources/guides/what-is-near-protocol_2.md:118]:
```suggestion
你需要确认两次交易，第一是选择验证者，第二是向验证者存入权益的数量
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+
+# 如何获得NEAR通证?
+有4种方式获得 $NEAR
+
+## 以太坊用户可获得免费的NEAR
+通过使用面向以太坊用户的水龙头[https://faucet.paras.id](https://faucet.paras.id)，用户可以得到一个拥有少量免费NEAR的钱包账户。
+
+## 赚取NEAR
+你可以通过参与[开发赏金任务](https://github.com/near/bounties)获得$NEAR, 运营一个帮助大家在NEAR上开发的[技术社区](https://near.org/guilds)，赢取NEAR骇客松比赛，或成为社区里的积极分子。如果你能吸引别人投NEAR给你抵押，你就可以成为一个验证人来赚取$NEAR。
```
Comment (think-in-universe) [inline cn/resources/guides/what-is-near-protocol_2.md:27]:
```suggestion
你可以通过参与[开发赏金任务](https://github.com/near/bounties)获得$NEAR, 运营一个帮助大家在NEAR上开发的[技术社区](https://near.org/guilds)，赢取NEAR黑客松比赛，或成为社区里的积极分子。如果你能吸引别人投NEAR给你抵押，你就可以成为一个验证人来赚取$NEAR。
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+你并不需要先有一个NEAR账户才能接收NEAR通证！“[NEAR Drop](https://near.org/blog/send-near-to-anyone-with-near-drops/)” 功能让你的朋友可以预付得到一个新账户并通过一个链接让你接收通证。
+
+# 我拿着NEAR通证能做什么？
+## 转账NEAR
+将 $NEAR 在你和朋友的账户之间转来转去，并可以在[区块浏览器](https://explorer.mainnet.near.org/)里查看对应的交易。因为交易手续费非常低，你可以方便快捷的转移非常小额的NEAR玩玩。
+
+把它们作为礼物通过[红包](http://redpacket.near.org/)送出去。
+
+## 试试NEAR空投 (邀请朋友)
+如果你的朋友向创建一个NEAR账户，可以给他们发送一个[NEAR空投](https://near.org/blog/send-near-to-anyone-with-near-drops/)。
```
Comment (think-in-universe) [inline cn/resources/guides/what-is-near-protocol_2.md:48]:
```suggestion
如果你的朋友想创建一个NEAR账户，可以给他们发送一个[NEAR空投](https://near.org/blog/send-near-to-anyone-with-near-drops/)。
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+与其他可编程网络类似，NEAR通过交易费，也叫Gas费补偿这些人。
+
+如果你熟悉web2上的云服务提供商（亚马逊云，谷歌云等），区块链与其的最大不同是，用户在访问应用的时候直接支付费用，而不是开发者预付所有的基础设施费用。这带来了新的可能性，例如，应用本身规避了长期的入不敷出的资金风险。但这也带来了一些用户可用性上的阻碍。为了解决这个问题，NEAR也提供了让开发者支付用户使用开销的能力，让使用体验更靠近web2。
+
+**考虑gas时要牢记的两个概念:**
+
+1. Gas单位: 在内部，交易费不是直接用NEAR通证计算的，而是通过一个居中的gas单位。gas单位的好处是，他们是确定的，相同的交易总是消耗相同数量的gas单位。
+2. Gas价格: gas单位乘以gas价格得到具体的向用户收取的费用。这个价格会在每个区块时根据网络需求自动重新计算（如果前面块的gas数量超过半满，则价格上升，否则价格下降，每个区块的价格变动不会超过1%）。最低的价格由网络配置决定，当前是1亿个yoctoNEAR。
+
+注意：测试网和装的gas价格可能不同。在直接使用下面的数字之前，要检查一下gas价格。
```
Comment (think-in-universe) [inline cn/resources/guides/what-is-near-protocol_2.md:69]:
```suggestion
注意：测试网和主网的gas价格可能不同。在直接使用下面的数字之前，要检查一下gas价格。
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 10]
```diff
@@ -0,0 +1,136 @@
...
+…这样一来，在最小1亿个yoctoNEAR每gas的价格下，等值于0.1个milliNEAR。
+
+这里的1ms是一个粗略但有用的近似值，也是目前NEAR中对gas单位设定的目标。Gas单位不仅封装了计算/CPU时间，也包含带宽/网络时间和存储/IO时间。通过治理机制，将来系统参数是可以调整的，可以改变TGas与1毫秒之间的对应关系。即便如此，上面的内容也是一个不错的开始，让你了解单位gas的意义以及它是怎么得来的。 
+
+**小测验: 1 TeraGas 或者说 TGas 是多少:**
+
+1 TeraGas或TGas等于:  
+1 微秒的计算时间  
+10 秒的计算时间  
+1 毫秒的计算时
```
Comment (think-in-universe) [inline cn/resources/guides/what-is-near-protocol_2.md:88]:
```suggestion
1 毫秒的计算时间
```

File: cn/resources/guides/what-is-near-protocol_2.md [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+
+这里的1ms是一个粗略但有用的近似值，也是目前NEAR中对gas单位设定的目标。Gas单位不仅封装了计算/CPU时间，也包含带宽/网络时间和存储/IO时间。通过治理机制，将来系统参数是可以调整的，可以改变TGas与1毫秒之间的对应关系。即便如此，上面的内容也是一个不错的开始，让你了解单位gas的意义以及它是怎么得来的。 
+
+**小测验: 1 TeraGas 或者说 TGas 是多少:**
+
+1 TeraGas或TGas等于:  
+1 微秒的计算时间  
+10 秒的计算时间  
+1 毫秒的计算时  
+1 纳秒的计算时
```
Comment (think-in-universe) [inline cn/resources/guides/what-is-near-protocol_2.md:89]:
```suggestion
1 纳秒的计算时间
```

**Review by think-in-universe — CHANGES_REQUESTED** [review]

**Comment by think-in-universe** [comment]
several typos that require change

**Comment by shepf** [comment]
翻译质量评价: 优秀

**Comment by wongxf** [comment]
翻译质量评价: 优秀

**Comment by think-in-universe** [comment]
翻译质量评价: 优秀

**Comment by MartinYeung5** [comment]
翻译质量评价: 优秀

**Pull request merged**